test/integration/podgc/podgc_test.go

```diff
@@ -56,6 +57,25 @@ func TestPodGcOrphanedPodsWithFinalizer(t *testing.T) {
 				Message: "PodGC: node no longer exists",
 			},
 		},
+		"PodDisruptionConditions and PodReplacementPolicy enabled": {
+			enablePodDisruptionConditions: true,
+			enablePodReplacementPolicy:    true,
```

Comment:
```suggestion
			enableJobPodReplacementPolicy:    true,
```
to make it clear. PodDisruptionConditions isn't actually scoped to just jobs.